V2/src/analogs.cpp

```diff
@@ -31,12 +31,12 @@ void analog_read_thread() {
     // Loop read adcs at 1kHz (does this actually run at 1 kHz?)
     while (true) {
         // todo - other sensors and scaling
-        analog_values.fcvolt = fcvolt.read() * 49.932f - 0.25f; //updated scaling factor
-        analog_values.motorvolt = motorvolt.read() * 26.70162f - 0.0357f;
-        analog_values.capvolt = capvolt.read() * 49.131f - 0.24f; //updated scaling factor
-        analog_values.fccurr = fccurr.read() * 157.7268f - 58.099f;
-        analog_values.motorcurr = motorcurr.read() * 315.4536f - 199.4556f;
-        analog_values.capcurr = capcurr.read() * 157.7268f - 58.099f;
+        analog_values.fcvolt = fcvolt.read() * 49.932f - 0.25f; //updated scaling factor, tested okay
+        analog_values.motorvolt = motorvolt.read() * 26.70162f - 0.0357f; //not working properly
+        analog_values.capvolt = capvolt.read() * 49.131f - 0.24f; //updated scaling factor, tested okay
+        analog_values.fccurr = fccurr.read() * 270.951f - 102.515f; //updated scaling factors, tested okay
+        analog_values.motorcurr = motorcurr.read() * 132.5529f - 54.3216f; //updated scaling factors, tested okay
+        analog_values.capcurr = capcurr.read() * 157.7268f - 58.099f; //not working properly
```

Comment:
I think there is an issue with reading this pin on the nucleo. Might have to jump it to another (Maybe one of the unused pressure sensors).

Reply:
Just made a new issue for this